docs/core/man/doveconf.1.md

```diff
@@ -26,10 +26,11 @@ dovecotComponent: core
 
 ## DESCRIPTION
 
-**doveconf** reads and parses Dovecot's configuration files and converts
-them into a simpler format used by the rest of Dovecot. All standalone
-programs, such as [[man,dovecot]], will first get their settings by executing
-doveconf.
+**doveconf** reads and parses Dovecot's configuration files and converts them
+into a simpler format used by the rest of Dovecot.
+
+All standalone programs, such as [[man,dovecot]], will first get their settings
+by executing doveconf.
```

Comment:
Hmm.. Since this is being changed, could have additional commit that continues this sentence: `by executing doveconf, unless they can get the settings by connecting to the config UNIX socket.`